andela/hc-leo
# 157694799 set timeout & grace for more than 30 days

**What does this PR do?**
This PR allows users to set timeouts and grace values for days exceeding 30.

**Description of Task to be completed?**
The dashboard only allowed setting grace period and timeouts to 30 days. This feature allows users to set the value to exceed 30days, that is, up to 1 year.

**How should this be manually tested?**
- Clone the repo, ```cd``` into the project and run ```./manage.py runserver```. 
- Navigate to the url provided by the development server (usually ```localhost```) using a browser.
- Create an account or log in
- Create a new check and set a suitable ```period``` and ```grace```.  The limit is now 1 year.


**What are relevant pivotal tracker stories?**
[#157694800](https://www.pivotaltracker.com/story/show/157694800)

**Relevant Screenshot**
<img width="580" alt="screen shot 2018-06-11 at 12 43 31" src="https://user-images.githubusercontent.com/24602451/41224483-43d5e804-6d75-11e8-866b-c1e776dabf9f.png">
